### PR TITLE
Curveball form

### PR DESCRIPTION
**Pull request by coati-00**
I opening a pull request from my fork to this branch so if you want to run it and see what I am doing you can. I change the facilitator dashboard to look like the one in the wireframes and created a new view for the new submit button on the curveball block.

### CODE REVIEW

File: media/js/uelc_admin/web_socks.js [hunk 1]
```diff
@@ -45,20 +45,25 @@ $(function() {
         var envelope = JSON.parse(evt.data);
         var data = JSON.parse(envelope.content);
 
-        //console.log("data");
-        //console.log(data);
+        console.log("data");
```
Comment (thraxil) [inline media/js/uelc_admin/web_socks.js]: stray `console.log()`s.

File: uelc/main/views.py [hunk 2]
```diff
@@ -156,13 +156,52 @@ def get_library_items(self, case):
...
+        '''if the notification is a quiz/decision submission
+        we need to get and return the curveball block'''
+        msg = dict()
+        if(notification == 'Decision Submitted'):
+            cb = self.section.get_next()
+            print "The next section is"
+            print cb
+            try:
+                if (cb.display_name == "Curveball Block"):
+                    print "at curveball"
```
Comment (thraxil) [inline uelc/main/views.py]: stray `print`

File: uelc/main/views.py [hunk 3]
```diff
@@ -156,13 +156,52 @@ def get_library_items(self, case):
...
+                        cb_two_title=cb.curveball_two.title,
+                        cb_two_explanation=cb.curveball_two.explanation,
+                        cb_three_title=cb.curveball_three.title,
+                        cb_three_explanation=cb.curveball_three.explanation)
+                    e = dict(address="%s.pages/%s/facilitator/" %
+                             (settings.ZMQ_APPNAME, self.section.hierarchy.name),
+                            content=json.dumps(msg))
+                    socket.send(json.dumps(e))
+                    socket.recv()
+            except:
```
Comment (thraxil) [inline uelc/main/views.py]: not a fan of bare `except`'s

**Comment by djredhand** [comment]
This fork is out of date.

**Pull request closed without merging**